src/services/doppler-billing-api-client.double.ts

```diff
@@ -170,21 +122,47 @@ export class HardcodedDopplerBillingApiClient implements DopplerBillingApiClient
           currency: 'ARS',
           amount: 2300,
           filename: 'invoice_2020-10-05_10.pdf',
-          _links: [
-            {
-              rel: 'help',
-              href: 'http://test.com/help',
-              description: 'Url to hel[',
-            },
-          ],
+          _links: [],
+        },
+        {
+          accountId: 'CD0000000073689',
+          product: 'Producto 11',
+          date: '2020-10-09T00:00:00',
+          currency: 'ARS',
+          amount: 2300,
+          filename: 'invoice_2020-10-05_10.pdf',
+          _links: [],
         },
       ],
-      totalItems: 10,
+      totalItems: 11,
     };
 
+    let invoicesPaged = [];
+
+    //The sort and paging is manually because it dummy service.
+    //For the real service, the sort and the paging is in the api
+    dataFromBilling.items.sort(function (a, b) {
```

Comment:
Why sort this here, and not have it hardcoded sorted? 🤔

Reply:
I thought to use an order funtion instead of to update the dummy data  because if in another moment the default order is another column is easier update the function that the dummy data. Do you prefer sort directly the data dammy?

Reply:
Si usamos generalmente los datos ordenados para evitar tener lógica en los dummies, pero no te veo convencido que esa forma de ordenar sea la que va, si decís que puede cambiar se podría dejar así.

Reply:
Joya, lo cambio no tengo ningun drama, yo me adapto a la forma de que trabajan ustedes, ahi ordeno los dummies y borro la funcion de orden. Muchas gracias por el feedback

Reply:
Done